### PR TITLE
PyLadies social event page

### PR DESCRIPTION
**Pull request by clytaemnestra**

### CODE REVIEW

File: data/pages-content/pyladies-social-event.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# PyLadies Social Event
+## About PyLadies
+PyLadies is an international mentorship group with focus on helping more women become active participants and leaders in the Python open-source community.
+Our mission is to promote, educate and advance a diverse Python community through outreach, education, conferences, events and social gatherings.
+
+## About the Event 
+The event is a fantastic opportunity to engage with other women in tech, expand your network, and share your experiences. Let's foster our community spirit, deepen connections, and spark new collaborations. Connect, share and celebrate!
+
+## Event @ EuroPython 
+- **When**: 28 July 2023
```
Comment (befeleme) [inline data/pages-content/pyladies-social-event.md:17]:
```suggestion
- **When**: 21 July 2023, start: 7 PM
```

File: data/pages-content/pyladies-social-event.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- **Where**: pub Hospůdka na Hradbách, 10 minutes by foot from Prague Congress Center
```
Comment (befeleme) [inline data/pages-content/pyladies-social-event.md:18]:
```suggestion
- **Where**: pub Hospůdka na Hradbách, 10 minutes on foot from Prague Congress Centre
```

File: data/pages-content/pyladies-social-event.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+ title: PyLadies Social Event 
+ subtitle: 
+   This year we're hosting a social event for PyLadies from Czech Republic and abroad. Let's connect, share and celebrate!
```
Comment (befeleme) [inline data/pages-content/pyladies-social-event.md:4]:
```suggestion
   This year we're hosting a social event for PyLadies from the Czech Republic and abroad. Let's connect, share and celebrate!
```

**Review by befeleme — CHANGES_REQUESTED** [review]
I like it! Some suggestions were proposed inline.
The date on the social card banner is wrong: it should be 21 July 2023.

What do you think about adding a form to gauge the interest from the community? I can prepare one and add the button - similarly to the way Snake workshop was done

**Comment by clytaemnestra** [comment]
> I like it! Some suggestions were proposed inline. The date on the social card banner is wrong: it should be 21 July 2023.
> 
> What do you think about adding a form to gauge the interest from the community? I can prepare one and add the button - similarly to the way Snake workshop was done

I remember we were discussing it, but I don't remember what was the conclusion. We definitely want to have it open for everyone even if they haven't registered. 

@hypha do you remember?

**Comment by hypha** [comment]
> > I like it! Some suggestions were proposed inline. The date on the social card banner is wrong: it should be 21 July 2023.
> > What do you think about adding a form to gauge the interest from the community? I can prepare one and add the button - similarly to the way Snake workshop was done
> 
> I remember we were discussing it, but I don't remember what was the conclusion. We definitely want to have it open for everyone even if they haven't registered.
> 
> @hypha do you remember?

Hey! I think we said that we could have a form to gauge interest but make it clear (on the form and website ) that this is mainly to help us with planning and doesn't mean they cannot join without signing up.

**Comment by clytaemnestra** [comment]
Okay, let's do it like that 🙂 So @befeleme let me know once the form is ready and I can update the page.

File: data/pages-content/pyladies-social-event.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+---  
+
+# PyLadies Social Event
+## About PyLadies
+PyLadies is an international mentorship group with focus on helping more women become active participants and leaders in the Python open-source community.
+Our mission is to promote, educate and advance a diverse Python community through outreach, education, conferences, events and social gatherings.
+
+## About the Event 
+The event is a fantastic opportunity to engage with other women in tech, expand your network, and share your experiences. Let's foster our community spirit, deepen connections, and spark new collaborations. Connect, share and celebrate!
+
```
Comment (befeleme) [inline data/pages-content/pyladies-social-event.md:15]:
```suggestion

<div style={{textAlign: "center", marginBottom: 20}}>
<ButtonLink href="https://forms.gle/Uz8Xamou6ik4mLEp6"> Register your interest now! </ButtonLink>
</div>

```

**Comment by befeleme** [comment]
@hypha, could you please rerun the deployment for visual control of the rendering? Thank you!

**Comment by hypha** [comment]
> @hypha, could you please rerun the deployment for visual control of the rendering? Thank you!
Of course! Sorry for the delay, @befeleme

**Review by befeleme — APPROVED** [review]
LGTM!

**Review by hypha — APPROVED** [review]
LGTM! Let me know when you want me to merge! <3

**Pull request merged**